pom.xml

```diff
@@ -51,10 +51,20 @@
   </dependencyManagement>
 
   <dependencies>
+
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-rx-java</artifactId>
```

Comment:
should be optional=true

Reply:
No, because the `VertxExtension` class needs the concrete types to operate.